src/index.ts

```diff
@@ -62,9 +64,28 @@ export default function (app: any) {
             lastMessages[0] = new Date().toISOString() + ':' + msg.nmea
             props.endpoints.forEach((endpoint: Endpoint) => {
               sendReportMsg(msg.nmea, endpoint.ipaddress, endpoint.port)
+
+              lastMsgNmea = msg.nmea
+              if (lastPositonTimeout) {
+                clearInterval(lastPositonTimeout)
+                lastPositonTimeout = undefined
+              }
+              if (props.lastpositonupdate) {
+                lastPositonTimeout = setInterval(
```

Comment:
You are creating an interval per each configured endpoint and then in each interval invocation sending the message to each endpoint. You should do either one or the other, this code sends n * n messages where n is the number of endpoints.